posts/2024-12-10_road-to-mainnet/README.md

```diff
@@ -0,0 +1,344 @@
+---
+publication_date: 2024-12-10T00:00:00Z
+slug: road-to-mainnet
+tags: [gnoland]
+authors: [adrianakalpa, michelleellen, thehowl, salmad3, Kouteki, leohhhn]
+---
+
+# The Road to Mainnet & Beyond
+
+## Gno the Difference
```

Comment:
I'm pretty sure this is not a "section" but a subtitle @salmad3

<img width="733" alt="Screenshot 2024-12-16 at 18 42 00" src="https://github.com/user-attachments/assets/1ef58e42-60c7-494a-88f3-4f61fd665153" />

Reply:
yep that was the intent